datafusion/sqllogictest/test_files/array.slt

```diff
@@ -2289,7 +2289,7 @@ select make_array(1.0, '2', null)
 ### FixedSizeListArray
 
 statement ok
-CREATE EXTERNAL TABLE fixed_size_list_array STORED AS PARQUET LOCATION 'tests/data/fixed_size_list_array.parquet';
+CREATE EXTERNAL TABLE fixed_size_list_array STORED AS PARQUET LOCATION '../core/tests/data/fixed_size_list_array.parquet';
```

Comment:
Eventually it might make sense to move the data files, but for now I just updated the test paths